python/tvm/micro/cross_compile.py

```diff
@@ -0,0 +1,48 @@
+"""Cross compilation for MicroTVM"""
+
+from __future__ import absolute_import
+
+import struct
+import logging
+import subprocess
+import os
+
+from .._ffi.function import _init_api
+from .._ffi.base import py_str
+
+
+def create_lib(output, sources, options=None, cc="gcc"):
```

Comment:
consider patch contrib.cc to introduce a cross compiler https://github.com/dmlc/tvm/blob/master/python/tvm/contrib/cc.py#L90 to move the fPIC and -shared to the cross_compiler part, so we can reuse these functions instead of creating our own.

Reply:
pending consolidation to cc

Reply:
I thought we were doing that in a separate PR?

Reply:
OK. that also works